ckanext/validation/tests/test_validators.py

```diff
@@ -6,6 +6,7 @@
 
 from ckan.tests.helpers import change_config
 
+import ckanext.validation.settings as settings
```

Comment:
Could be written more simply as `from ckanext.validation import settings`.

Reply:
\+